packages/flutter_driver/lib/src/driver.dart

```diff
@@ -4,20 +4,32 @@
 
 import 'dart:async';
 import 'package:vm_service_client/vm_service_client.dart';
+import 'package:matcher/matcher.dart';
```

Comment:
We should add `matcher` to the pubspec; I think it's being brought in now as a transitive dep.